.github/workflows/be-deploy.yml

```diff
@@ -1,6 +1,8 @@
 name: Deploy backend (main)
 
 on:
+  pull_request:
+    branches: [main]
```

Comment:
😂 that explains a lot

Reply:
I don't know what do you mean. But it's removed now

Reply:
It explained why the CI wasn't running on branch PR.